tracer/build/_build/Build.Steps.cs

```diff
@@ -1828,7 +1867,124 @@ async Task RunWindowsIisIntegrationTests(Project project)
             CopyDirectoryRecursively(MonitoringHomeDirectory, target, DirectoryExistsPolicy.Merge, FileExistsPolicy.Overwrite);
         });
 
+    Target CreateRootDescriptorsFile => _ => _
+       .Description("Create RootDescriptors.xml file")
+       .DependsOn(CompileManagedSrc)
+       .Executes(() =>
+        {
+            var datadogTraceDirectory = Solution.GetProject(Projects.DatadogTrace).Directory;
+            var loaderTypes = GetTypeReferences(SourceDirectory / "bin" / "ProfilerResources" / "netcoreapp2.0" / "Datadog.Trace.ClrProfiler.Managed.Loader.dll");
+            var tracer3_1Types = GetTypeReferences(datadogTraceDirectory / "bin" / BuildConfiguration / TargetFramework.NETCOREAPP3_1 / Projects.DatadogTrace + ".dll");
+            var tracer6_0Types = GetTypeReferences(datadogTraceDirectory / "bin" / BuildConfiguration / TargetFramework.NET6_0 / Projects.DatadogTrace + ".dll");
```

Comment:
Kind of surprising we haven't defined these paths somewhere... I bet this one will get missed if we ever change the TFMs we target 😬

Reply:
@tonyredondo if this path gets messed up, will we fail in a really obvious way? If we do, that would help catch any regressions before shipping

Reply:
Yes, if the file is not found the `Mono.Cecil.AssemblyDefinition.ReadAssembly` will fail. Although, we might miss new assemblies if we add new TFMs in the future. Let me try to leverage this to a TFMs array or something.

Reply:
Improved here: [3f7d8c1](https://github.com/DataDog/dd-trace-dotnet/pull/4195/commits/3f7d8c1203bfd826fbbc0322bb1e81d06c4a0b71)